news/latest/2016.01.02.08.06.1.story

```diff
@@ -0,0 +1,40 @@
+@@title:
+KVIrc 4.3.2 Dev Progress, The Road to Next 5.x Stable Nears.
+@@short:
+KVIrc Dev News
+@@author:
+un1versal
+@@body:
+
+
+<p>
+Since moving to GitHub, a great deal has happened to the KVIrc you know and love.
+Change and progress is inevitable and as with all things, some people find changes harder to get used to than others, but here is to hoping you like KVIrc as much as ever, smiley-face.
+<p>
+Notably, KVIrc has now a new complete set of graphics and program icons and a few new icons have been added and some removed as a result.<br>
+While the old icon set was replaced, its certainly not gone, for the die-hard fans who find change disconcerting,
+they can still find it at its new home <a href="https://github.com/kvirc/kvirc-themes ">at KVIrc themes GitHub repo</a>.<br>
+Theres has been much progress in this area but much cleanup remains to be done as a result of the many changes KVIrc has undergone, We have said our goodbyes to KviMdiManager and welcomed KviWindowStack.
+Icons have been dropped due to change and others have been renamed, so this also means that all old themes will be look slightly odd since when new icons don't exist in a theme, the core ones are shown.
+<br></p><p>
+We continue to work really hard in improving the look and consistency of KVIrc, and hope to have a new stable for you soon. In the meanwhile if you're on Windows, Debian or Ubuntu please visit the <a href="https://github.com/kvirc/KVIrc/wiki/Downloading-KVIrc's-nightly-source-or-binaries">Our GitHub downloads page</a> and grab a nightly build to use and test, its quite save and stable to use daily.<br><br> 
+One of those changes are the core strings and help texts which have been and are still undergoing and ongoing revision.<br>
+It is our hope our users wont stumble into quite so many typos (even if the odd one slipped through), but also our hard working translators that will hopefully not have to work around issues with core strings in their translations anymore.
+Many string consistencies have been addressed, its possible some have managed to slip through though, but we hope to squash them all eventually.<br>
+We certainly would appreciate that our translators/users report any issues at our <a href="https://github.com/kvirc/KVIrc/issues">KVIrc issue tracker</a>, like broken syntax a.k.a. broken English or any other issue, so we can instead of working around it in translations, fix t in core instead.
+<br></p><p>
+While Im personally not a fan of stats and the stats bellow at are missing some data thats not accounted for, the move to GitHub makes collating these changes a much simpler process; here are some stats you may find interesting:
+</p><li><a href="https://github.com/kvirc/KVIrc/commit/6c7203936f98e782c23e365ebc368919c4623004">6000 commits</a> have made its way into Master since moving to GitHub (100% accurate at time or writing)
```

Comment:
You're confusing 2 different numbers.
Since beginning of times it's 6000.
Since moving to Github it's 911.